easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.6.0.eb

```diff
@@ -0,0 +1,36 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'Tarball'
+
+name = 'Trim_Galore'
+version = '0.6.0'
+
+homepage = 'https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """Trim Galore! is a wrapper script to automate quality and 
+adapter trimming as well as quality control, with some added functionality 
+to remove biased methylation positions for RRBS sequence files (for directional, 
+non-directional (or paired-end) sequencing)."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/FelixKrueger/TrimGalore/archive/']
+sources = [{
+    'filename': '39e1ad7.tar.gz',
+}]
+
+checksums = ['aad008ff2e9605079655013bd91615ca2272048b151b2fe7ee197d18b1b87908']
+
+dependencies = [
+    ('FastQC', '0.11.8-Java-1.8'),
+    ('cutadapt', '1.18-foss-2018b-Python-2.7.15')
```

Comment:
Trim_Galore actually depends on Perl and needs a patch to not use perl from /usr/bin/perl but instead
#!/usr/bin/env perl
And thus don't really belong in dummy, unless an explicit osdependency for the respecitve perl packages for debian/RH is added.

Reply:
And, since I was slightly blind last time i looked at this...
that cutadap dependency is wrong.
Trim_Galore must use the foss/2018b toolchain so it can depend on cutadapt correctly.
And further more, w already have Trim_Galore-0.6.0-foss-2018b-Python-2.7.15.eb merged so this PR is likely not needed any longer.
@sassy-crick Close it?

Reply:
Hi akesandgren,
thanks for the feedback and sorry for my slow response. I have done the requested changes and created a patch-file for the Perl issue. I have noticed there is a newer version out by now. If you are happy with this EB build file, I would suggest we can close this one and I upload the EB build file for the new version. Are you happy with this?

Reply:
@akesandgren 
The same issue regarding Perl was also mentioned here:
https://github.com/easybuilders/easybuild-easyconfigs/pull/8019
How shall we proceed regarding Perl in Trim_Galore?
As I said, I have no opinion here at all.

Reply:
@sassy-crick In #7893, it's dealt with via a `sed` command, which is more portable w.r.t. maintaining easyconfigs for future `Trim_Galore` versions...